### PR TITLE
Realtime

### LINKED ISSUE #25: real-time congestion controls
Condition (1) is concerned with the behavior with respect to Reno and, thanks to our edits, Cubic. But there are of course other congestion controls out there, both standard and non-standard, in support of real-time and other non-greedy applications. Should we mention those as a class? Namecheck some examples?

### PR DESCRIPTION
**Pull request by martinduke**
Fix #25 and #32.

### CODE REVIEW

**Comment by martinduke** [comment]
Also #35

**Review by huitema — APPROVED** [review]
Looks good enough.

File: draft-ietf-ccwg-rfc5033bis.md [hunk 1]
```diff
@@ -395,8 +395,28 @@ discussed in Sections 6, 11.1, and 12 of {{?RFC3649}}.
...
+detailed public specifications are not widely deployed, while many widely
+deployed real-time congestion controls have incomplete public specifications.
+
+To the extent behavior of widely deployed algorithms is understood, proposals
+can analyze and simulate the their interaction with those algorithms. To the
+extent they are not, experiments can be conducted where possible.
+
+Note that in many deployments, real-time traffic is directed into distinct
+queues via Differentiated Services Code Points (DSCP) or other mechanisms,
+which substantially reduces the interplay between algorithms. However, proposals
```
Comment (gorryfair) [inline draft-ietf-ccwg-rfc5033bis.md:418]: Interplay between algorithms - or interplay with other traffic : I'd have thought the latter?

File: draft-ietf-ccwg-rfc5033bis.md [hunk 2]
```diff
@@ -395,8 +395,28 @@ discussed in Sections 6, 11.1, and 12 of {{?RFC3649}}.
 
 ### Real-Time Protocols
 
-(TODO: Clarify that real time congestion controls are included, with
-allowances for the poor documentation / open source availability of these)
+General-purpose protocols coexist in the internet with real-time congestion
+control protocols, which in general have finite throughput requirements and
+more strict latency bounds.
```
Comment (gorryfair) [inline draft-ietf-ccwg-rfc5033bis.md:400]: These protocols are often not capacity seeking, and hence methods such as CWV might apply [RFC7661]?

File: draft-ietf-ccwg-rfc5033bis.md [hunk 3]
```diff
@@ -395,8 +395,28 @@ discussed in Sections 6, 11.1, and 12 of {{?RFC3649}}.
...
+
+{{?RFC8868}} provides suggestions for real-time congestion control design and
+{{?RFC8867}} suggests test cases. {{?RFC9392}} descfibees some considerations
+for the RTP Control Protocol (RTCP). This document does not change the
+informational status of those RFCs.
+
+New proposals SHOULD consider coexistence with widely deployed real-time
+congestion controls. Regrettably, at the time of writing, many algorithms with
```
Comment (gorryfair) [inline draft-ietf-ccwg-rfc5033bis.md:408]: Why not with widely-deploted transports in general here? Why specific to real-time protocols?

**Review by gorryfair — COMMENTED** [review]
Some thoughts...

File: draft-ietf-ccwg-rfc5033bis.md (hunk 3 above)
Comment (martinduke) [reply draft-ietf-ccwg-rfc5033bis.md:408]: well, the section is about real-time transports. General purpose protocols are in the section above.

File: draft-ietf-ccwg-rfc5033bis.md (hunk 2 above)
Comment (martinduke) [reply draft-ietf-ccwg-rfc5033bis.md:400]: "finite throughput requirements" was my way of saying "non capacity seeking".  I don't think it's necessarily the role of this document to identify examples of specific congestion control proposals.

Comment (gorryfair) [reply draft-ietf-ccwg-rfc5033bis.md:400]: Sure, we ought not to advocate methods... but I think  we could day "(i.e. does not seek to utilise all allowed capacity)" or something to indicate what we mean

File: draft-ietf-ccwg-rfc5033bis.md (hunk 3 above)
Comment (gorryfair) [reply draft-ietf-ccwg-rfc5033bis.md:409]: Sorry - not my point. I meant: Why not "New real-time congestion control proposals SHOULD consider coexistence with widely deployed real-time and non-real-time congestion controls."

File: draft-ietf-ccwg-rfc5033bis.md [hunk 4]
```diff
@@ -395,8 +395,29 @@ discussed in Sections 6, 11.1, and 12 of {{?RFC3649}}.
...
+General-purpose protocols coexist in the Internet with real-time congestion
+control protocols, which in general have finite throughput requirements (i.e.
+they do not seek to utilize all available capacity) and more strict latency
+bounds.
+
+{{?RFC8868}} provides suggestions for real-time congestion control design and
+{{?RFC8867}} suggests test cases. {{?RFC9392}} describes some considerations
+for the RTP Control Protocol (RTCP). This document does not change the
+informational status of those RFCs.
+
```
Comment (gorryfair) [inline draft-ietf-ccwg-rfc5033bis.md:407]: Feedback for congestin control  for real-time flows can be diifferent and less frequent than the acknowledgements provided by reliable transports, RFC 9392 provides some guidance on how to design and evaluate appropriate feedback.

File: draft-ietf-ccwg-rfc5033bis.md (hunk 3 above)
Comment (martinduke) [reply draft-ietf-ccwg-rfc5033bis.md:409]: Editors meeting: this doc is saying all algs (of any type) must be tested against cubic, etc AND real-time protocols. There are not different standards for different types of proposal.

**Review by gorryfair — APPROVED** [review]
Thanks for the changes.

**Pull request merged**